knapsack/branch_and_bound/item.py

```diff
@@ -0,0 +1,38 @@
+class Item:
+    def __init__(self, name: str, weight: float, rating: float) -> None:
+        self._name = name
+        self._weight = weight
+        self._rating = rating
```

Comment:
`rating`means `value` I guess? See if we can align it with the definition in README.